www/install/sql/centreon/Update-DB-21.04.2.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM `topology` WHERE `topology_page` IN (6090901, 6090902);
```

Comment:
won't be backportable on 20.10 and lesser versions
you may need to split this PR by maintenance version

Reply:
will be handled while cherry-picking